packages/kbn-es-archiver/src/actions/empty_kibana_index.ts

```diff
@@ -26,6 +26,7 @@ export async function emptyKibanaIndexAction({
 
   await cleanSavedObjectIndices({ client, stats, log });
   await migrateSavedObjectIndices(kbnClient);
+  await client.indices.refresh({ index: ALL_SAVED_OBJECT_INDICES });
```

Comment:
One test was failing without the implicit refresh in-between tests. I think it makes sense to add this refresh as part of `emptyKibanIndex` see for more details https://github.com/elastic/kibana/issues/159662#issuecomment-1621441238

Reply:
I would ask @gsoldevila opinion here since here recently did a great job reviewing archiver logic.
If it is failing for the particular test, is it possible to explicitly refresh indexes in that test?

Reply:
@dmlemeshko, see [here failure number 2 ](https://github.com/elastic/kibana/issues/159662#issuecomment-1621441238). 

I think it is possible to add a refresh inside that specific test, but it seemed it made more sense to refresh inside esArchiver.emptyKibanaIndex

Reply:
Yep, it totally makes sense to have the refresh with this action as well.
For comparison, _esArchiver_'s `loadAction` does the same thing already.

Reply:
Thanks for the explanation, I'm good with code change.